integration/main.browser.ts

```diff
@@ -13,7 +13,7 @@ const bootstrap = () => platformBrowserDynamic().bootstrapModule(AppBrowserModul
 
 if (environment.hmr) {
   if (module['hot']) {
-    hmrNgxsBootstrap(module, bootstrap).catch(err => console.log(err));
+    hmr({ module, bootstrap }).catch(err => console.error(err));
```

Comment:
Since module and bootstrap are not optional parameters and are critical for the hmr plugin to work these should not be part of the options but rather be the first 2 compulsory parameters.

Reply:
done